liquibase-standard/src/main/java/liquibase/serializer/core/formattedsql/FormattedSqlChangeLogSerializer.java

```diff
@@ -88,6 +88,12 @@ public void createChangeSetInfo(ChangeSet changeSet, StringBuilder builder) {
             builder.append(outputContexts);
             builder.append("\"");
         }
+        String logicalFilePath = changeSet.getLogicalFilePath();
+        if (logicalFilePath != null && ! logicalFilePath.isEmpty()) {
+            builder.append(" logicalFilePath: \"");
```

Comment:
Is there any specific reason why space is being added after `:` for this property? If I'm not wrong, all the values for SQL format properties are specified next to `:`.

Reply:
I added the space just because the other properties above also had the space, and I didn't want to deviate from a convention.  I agree it seems a little weird, happy to remove the space from all of them (or just this one) if you'd like.

Reply:
My bad, you are right that other properties look the same way. I was looking at doc [examples](https://docs.liquibase.com/concepts/changelogs/home.html) (SQL example) and ask you without looking at the other attributes in the code.

My apologies.

Reply:
No problem at all!  I had the exact same reaction when I first saw the code - and still wonder whether it's "proper".

Reply:
I'll check this with our product team and according to their answer we might need to update docs.

Thanks,
Daniel.

Reply:
I was just doing a bit of manual testing and it seems docs are right and the code is wrong. If we have a changeset header like the one below:

```sql
--changeset mallodTest:1 labels: test
```

and then at runtime, we specify: liquibase update --labels="test2". Changeset is processed anyway and deployed. 

Now if it is:

```sql
--changeset mallodTest:1 labels: test
```

Label specified in the command has be an exact match to make that changeset processed.

Reply:
Gotcha, I'll update the code here.